docs/content/any/project/changelogs/2021-06-22.md

```diff
@@ -50,6 +50,40 @@ allow(_user, action, repository: Repository) if
 
 ##### Built-in Role-Based Access Control (RBAC) policy
 
+Oso's Python and Node libraries now include [built-in support for Role-Based Access Control (RBAC) policies](/guides/roles).
+The goal of introducing this feature is to make setting up common role-based access control patterns even easier
+by providing a structured configuration interface and implementing the query evaluation logic.
+
+Here's an example policy using the new configuration interface:
+
+```polar
+resource(_type: Org, "org", actions, roles) if
+    actions = ["read", "create_repo"] and
+    roles = {
+        member: {
+            permissions: ["read"],
+        },
+        owner: {
+            permissions: ["read", "create_repo"],
+        }
+    };
+```
+
+The configuration interface allows Oso users to:
+- Define **resource types** and **roles** that are used to control access to individual resources of that type (resource-specific roles)
+  - E.g., users can define `Organization` resources that have `owner`, `member`, and `billing` roles.
+- Define the **actions** that each role allows users to take
+  - E.g., the `Organization` `owner` role allows users to take the `invite_member` action
+- Define parent-child relationships between resources to enable features like **implied roles and nested permissions**
+  - E.g., if a user has the `owner` role on an organization, they also have the `admin` role implied for every child repository of the organization
+
+In order to use the built-in policy, Oso users must store user-role assignments themselves and implement a simple interface to expose the user-role assignment data to Oso's policy engine.
+
+Then, the role-based policy will be evaluated as any other policy would by calling the `is_allowed()` library method.
+
+To get started, check out [the docs](/guides/roles).
+
+
 <!-- TODO(gj): usual caveat about this being an experimental feature; talk to us
 if you want to use it in prod. -->
```

Comment:
should we do this? cc: @samscott89

Reply:
I also have a question about whether we want to address that data filtering isn't supported here?

Reply:
> I also have a question about whether we want to address that data filtering isn't supported here?

I think it's fine to leave the caveat out of here since it exists in the Django & SQLAlchemy docs.